subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -415,6 +416,13 @@ int ll_init(struct k_sem *sem_rx)
 		ull_filter_reset(true);
 	}
 
+#if IS_ENABLED(CONFIG_BT_CTLR_DF)
+	err = lll_df_init();
+	if (err) {
+		return err;
+	}
+#endif
+
```

Comment:
Move to line 406 after BT_CONN and before USER_EXT

Reply:
Moved to requested line.